### ISSUE TITLE
Anaconda new license - and bioconda packages

### ISSUE THREAD

**Issue by ypriverol** [bioconda/bioconda-recipes#49877]
Hi everyone,

Our institution has recently been impacted by the new Anaconda license, and although we are an academic organization, it seems we may no longer have access to the service. I'm curious to know if others in the community are also affected by this change, and whether Bioconda as a community might be able to suggest a possible solution. After all, aren't all Bioconda packages hosted on Anaconda.org?

Best regards,
Yasset

**Comment by ebete** [comment]
I think this mostly applies on the anaconda repository (https://anaconda.org/anaconda) and not on the `conda-forge` and `bioconda` repos. In the ToS they only explicitly say the "Anaconda Public Repository" is covered by the ToS.

Anaconda ToS §2.1

>Utilizing Miniconda to pull package updates from the Anaconda Public Repository without a commercial license (if required by the conditions set forth in Section 2 of this Terms of Service) is considered a violation of the Terms of Service.

**Comment by ypriverol** [comment]
It looks like it is more complex than that, it could affect other repos and URLs, for example, [anaconda.org/r](http://anaconda.org/r)

**Comment by ebete** [comment]
Yes because that's an Anaconda repo as well.

https://repo.anaconda.com/pkgs/ shows some more like:

- `main` (part of `defaults`)
- `free`
- `r` (part of `defaults`)
- `mro`
- `pro`
- `archive`
- `mro-archive`
- `msys2` (part of `defaults`)

**Comment by peterjc** [comment]
I/we have started deliberately moving from miniconda (with non-free defaults channel) to miniforge (with conda-forge as the only channel) as our base conda, to which we then add bioconda - https://github.com/conda-forge/miniforge

**Comment by corneliusroemer** [comment]
TL;DR: Just don't use the anaconda channels, e.g. by using miniforge/mambaforge or excluding default channels - then there's no problem whatsoever.

The issue has been discussed already at length back in the days when Anaconda first started restricting TOS.

Here are a few links for further reading:
- https://www.anaconda.com/blog/is-conda-free
- https://stackoverflow.com/questions/65256884/is-an-anaconda-commerical-edition-license-required-for-conda-forge-repositories
- https://conda-forge.org/blog/2020/11/20/anaconda-tos/
- https://www.reddit.com/r/Python/comments/iqsk3y/anaconda_is_not_free_for_commercial_use_anymore/
- https://florianwilhelm.info/2021/09/Handling_Anaconda_without_getting_constricted/
